File: app/src/main/java/br/com/jonathanarodr/playmovie/ui/MovieReviewAdapter.java
package br.com.jonathanarodr.playmovie.ui;

import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.util.List;

import br.com.jonathanarodr.playmovie.R;
import br.com.jonathanarodr.playmovie.model.MovieReview;

public class MovieReviewAdapter extends RecyclerView.Adapter<MovieReviewAdapter.MovieReviewViewHolder> {

    private List<MovieReview> mReviews;

    public void setReviews(List<MovieReview> reviews) {
        this.mReviews = reviews;
        notifyDataSetChanged();
    }

    class MovieReviewViewHolder extends RecyclerView.ViewHolder {

        private final TextView mAuthor;
        private final TextView mContent;

        MovieReviewViewHolder(View itemView) {
            super(itemView);
            mAuthor = itemView.findViewById(R.id.author_review);
            mContent = itemView.findViewById(R.id.content_review);
        }

    }

    @NonNull
    @Override
    public MovieReviewViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_review, parent, false);
        return new MovieReviewViewHolder(view);
    }

    @Override
    public void onBindViewHolder(@NonNull MovieReviewViewHolder holder, int position) {
        MovieReview review = mReviews.get(position);
        holder.mAuthor.setText(review.getAuthor().trim());
        holder.mContent.setText(review.getContent().trim());
    }

    @Override
    public int getItemCount() {
        return (mReviews == null) ? 0 : mReviews.size();
    }

}

File: app/src/main/java/br/com/jonathanarodr/playmovie/ui/MovieReviewAdapter.kt
package br.com.jonathanarodr.playmovie.ui

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView.Adapter
import androidx.recyclerview.widget.RecyclerView.ViewHolder
import br.com.jonathanarodr.playmovie.R.id
import br.com.jonathanarodr.playmovie.R.layout
import br.com.jonathanarodr.playmovie.model.MovieReview

class MovieReviewAdapter : Adapter<MovieReviewAdapter.MovieReviewViewHolder>() {

    var reviews: List<MovieReview> = emptyList()
        set(value) {
            field = value
            notifyDataSetChanged()
        }

    class MovieReviewViewHolder(itemView: View) : ViewHolder(itemView) {
        val author: TextView = itemView.findViewById(id.author_review)
        val content: TextView = itemView.findViewById(id.content_review)
    }

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieReviewViewHolder {
        val view: View = LayoutInflater.from(parent.context)
            .inflate(layout.item_review, parent, false)

        return MovieReviewViewHolder(view)
    }

    override fun onBindViewHolder(holder: MovieReviewViewHolder, position: Int) {
        val (_, author, content) = reviews[position]
        holder.author.text = author.trim()
        holder.content.text = content.trim()
    }

    override fun getItemCount(): Int {
        return reviews.size
    }
}